students/gangho/users/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
 
-from users.views import SignupView
+from users.views import SignupView, SigninView
 
 urlpatterns = [
-    path('/signup', SignupView.as_view())
+    path('/signup', SignupView.as_view()),
+    path('/login', SigninView.as_view())
```

Comment:
`signin`이면 엔드포인트 주소도 똑같이 `signin`으로 해주시는게 좋겠죠?
푸쉬전에 전체적으로 코드를 점검해주세요!
```suggestion
    path('/sigjnin', SigninView.as_view())
```